tests/unit/s2n_self_talk_session_resumption_test.c

```diff
@@ -498,22 +498,26 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_io_pair_close(&io_pair));
     }
 
-    /* Test: A TLS1.3 client with a valid TLS1.2 ticket can fall back to a TLS1.3 connection
+    /* Test: A TLS1.3 client with a valid TLS1.2 ticket can fall back to a
+     * TLS1.3 connection.
+     * Test: When TLS 1.3 is negotiated s2n_connection_get_session should not
+     * return a ticket before it has been received.
```

Comment:
I'd remove this from the main test name. It's more of an assertion you make about the specific self-talk scenario you're testing. We have unit tests that test "should not return a ticket before it has been received" for more normal scenarios.